lib/index.js

```diff
@@ -55,6 +55,7 @@ internals.schema = Joi.object({
     ttl: Joi.number(),
     domain: Joi.string().allow(null),
     providerParams: Joi.object(),
+    allowRuntimeQueryParams: Joi.boolean().default(false),
```

Comment:
We've only implemented it for oauth2 so adding `.when('provider.protocol', { is: 'oauth2', otherwise: Joi.forbidden() })` might be useful so people don't wonder why it is not working.